README.md

```diff
@@ -20,6 +20,10 @@ configuration across the remaining aggregators in a group
 In this current release, Maestro does not start and start __ldmsd__
 daemons, however, this feature is planned for the future.
 
+## Dependencies
+python-hostlist
+etcd3
```

Comment:
Is this the actual package name?

Reply:
Yes, these are the actual package names as installed with Python3 pip (pip3).

Reply:
@nick-enoent, can you remove the python-hostlist dependency from the README? Also add a source for the etcd3 code.

Reply:
Sure thing